Documentation/Plugins.md

```diff
@@ -203,13 +207,9 @@ struct MyBuildToolPlugin: BuildToolPlugin {
 }
 ```
 
-In the case of prebuild commands, any dependencies must be binary targets, since these commands run before the build starts.
-
-Note that a build tool plugin can return a combination of build tool commands and prebuild commands.  After the plugin runs, any build commands are incorporated into the build graph, which may result in changes that require commands to run during the subsequent build.
-
-Any prebuild commands are run after the plugin runs but before the build starts, and any files that are in the prebuild command's declared `outputFilesDirectory` will be evaluated as if they had been source files in the target.  The prebuild command should add or remove files in this directory to reflect the results of having run the command.
+A prebuild command has no inputs, so it will never be re-run due to changes in source files. The only trigger for re-running a prebuild command is a change to a declared dependendencies, which can only be (prebuilt) binary targets, since these commands run before any other targets have been built.
```

Comment:
typo nit
```suggestion
A prebuild command has no inputs, so it will never be re-run due to changes in source files. The only trigger for re-running a prebuild command is a change to declared dependencies, which can only be (prebuilt) binary targets, since these commands run before any other targets have been built.
```